aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -781,6 +781,17 @@ void TensorIterator::cast_outputs() {
   }
 }
 
+void TensorIterator::copy_reduction_outputs(){
```

Comment:
nit: don't we have reductions that only have a single output?  We use "outputs" above to mean "1 or more" but here it means "more than 1".  Maybe we can come up with a term for this?  Like `copy_reduction_multi_outputs` or something.